tests/basic/storage_pool_test.go

```diff
@@ -9894,3 +9898,219 @@ var _ = Describe("{AddDriveWithKernelPanic}", func() {
 	})
 
 })
+
+func GetNumOfDrivesInNode(stNode node.Node) int {
+	numofDrivesInNode := 0
+	poolListForOps, err := GetPoolsDetailsOnNode(stNode)
+	log.FailOnError(err, "failed while getting updated node pool list")
```

Comment:
return the err here and move this to common

Reply:
Done